### PR TITLE
Removed appraise for 1.x

### PR DESCRIPTION
**Pull request by Anu-MT**
This PR is for:
Upgrade minimal gemspec dependencies to Sinatra > 2
Issue:  https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/364

### CODE REVIEW

File: instrumentation/sinatra/Appraisals [hunk 1]
```diff
@@ -12,6 +12,4 @@ appraise 'sinatra-2.x' do
   gem 'sinatra', '~> 2.1'
 end
 
-appraise 'sinatra-1.x' do
-  gem 'sinatra', '~> 1.4'
-end
+
```
Comment (ericmustin) [inline instrumentation/sinatra/Appraisals:15]:
```suggestion
```

**Review by ericmustin — CHANGES_REQUESTED** [review]

**Comment by ericmustin** [comment]
@Anu-MT looks like just a quick rubocop error then shud be good to merge, ty for the contribution

**Comment by arielvalentin** [comment]
@Anu-MT will you also be flowing up to update the minimum version requirements for the instrumentation itself?

**Comment by Anu-MT** [comment]
Hi,

Yes, I would like to work on that as well. Will do it within this week.

Thanks,
Anu

**Pull request closed without merging**